devel/docker/settings/greenwave-settings.py

```diff
@@ -0,0 +1,16 @@
+HOST = '0.0.0.0'
+PORT = 6545
+DEBUG = False
+POLICIES_DIR = '/etc/greenwave/policies/'
+DIST_GIT_BASE_URL = 'https://src.fedoraproject.org'
+DIST_GIT_URL_TEMPLATE = '{DIST_GIT_BASE_URL}/{pkg_namespace}/{pkg_name}/raw/{rev}/f/gating.yaml'
+KOJI_BASE_URL = 'https://koji.fedoraproject.org/kojihub'
+BODHI_URL = 'https://bodhi/'
```

Comment:
Should this be:

```suggestion
BODHI_URL = 'http://localhost:6543/'
```

?

Reply:
Copy-paste from CI. But now I see that this setting is not used in GW. Removing it.

Reply:
Looks to me like this is resolved.